reactioncommerce/reaction
# Add to Cart Alert Behavior

Resolves [1445](https://github.com/reactioncommerce/reaction/issues/1445) and [2042](https://github.com/reactioncommerce/reaction/issues/2042)

***Testing***

- Clone this branch and start up `reaction`
- Select a product and click `add to cart` button rapidly
- Note the spinner on rapid clicks
- Check that the cart drawer behaves the same way that it used to
